README.md

```diff
@@ -393,6 +393,18 @@ Supported platforms: GNU/Linux, FreeBSD.
     `${down_b}`, `${up_b}`, `${down_kb}`, `${up_kb}`, `${down_mb}`, `${up_mb}`,
     `${down_gb}`, `${up_gb}`.
 
+### vicious.widgets.notmuch
+
+Provides a message count according to an arbitrary Notmuch query.
+
+Supported platforms: platform independent.
+
+Argument: the query that is passed to Notmuch. For instance: `tag:inbox AND tag:unread` returns the number of unread messages with tag "inbox". 
```

Comment:
I notice a trailing space at the end of this line and it's also bit long(er than the 80 column limit).

Reply:
I've reformatted that paragraph. Cheers.